_topic_maps/_topic_map.yml

```
@@ -142,6 +142,8 @@ Topics:
       File: installing-alibaba-customizations
     - Name: Installing a cluster on Alibaba Cloud with network customizations
       File: installing-alibaba-network-customizations
+    - Name: Installing a cluster on Alibaba Cloud with VPC customizations
```

```suggestion
    - Name: Installing a cluster on Alibaba Cloud into a shared VPC
```
I'm suggesting this because it's how we named the GCP shared VPC installation assembly. Up to you.

---

@bscott-rh Thank you! Done!